configs/sst_dotnet-dotnet.yaml

```diff
@@ -2,24 +2,14 @@ document: feedback-pipeline-workload
 version: 1
 data:
   name: .NET
-  description: .NET and .NET Core SDK, tools and runtime
+  description: .NET SDK, tools and runtime
   maintainer: sst_dotnet
 
-  packages: []
-  
-  arch_packages:
```

Comment:
.NET 7 and later are available for all 4 architectures that Fedora/RHEL care about.